test/gigs/5111.1.gie

```diff
@@ -21,7 +21,7 @@ tolerance  0.05 m
 accept    100.0876483 77.6534822
 expect    2800000.0 15000000.0
 
-tolerance  0.05 m
+tolerance  0.055 m
```

Comment:
Why is the extra 5 mm needed here?

Reply:
To make the test pass... I can't make sense about the towgs84=0,0,0 default in the pipeline definition

Reply:
I think the idea is to override the `+towgs84` parameters in the epsg init file, probably in an effort to work around `towgs84=0,0,0` not being handled correctly before.

It makes sense that you have to change the tolerance since `towgs84` is now handled differently.